packages/lesswrong/components/review/ReviewVotingPage.tsx

```diff
@@ -509,7 +509,7 @@ const ReviewVotingPage = ({classes}: {
 }
 
 function getPostOrder(posts: Array<PostsList>, votes: Array<qualitativeVote|quadraticVote>, currentUser: UsersCurrent|null): Array<[number,number]> {
```

Comment:
Should the function argument be `posts: Array<PostsList> | undefined`? That would have allowed you to catch the error with TS earlier, and seems like a more correct expression of things. Also raises the question of whether there's an upstream function that says it returns a `PostsList[]` but in fact returns `PostsList | undefined`.